pkg/persistence/config/loader/config_entry_loader.go

```diff
@@ -70,6 +74,8 @@ func parseConfigEntry(
 			continue
 		}
 
+		report.GetReporterFromContextOrDiscard(ctx).ReportLoading(report.StateSuccess, nil, "config validated", &result.Coordinate)
```

Comment:
I'd add a basic test that if a config is loaded, the report is written to.

Reply:
In `loading-report_test.go` there is actually a test that validates the success loading event